README.md

```diff
@@ -128,7 +128,118 @@ the graphical user interface.
 
 **under construction**
 
+## Tutorial for parsing results obtained with SCIP
 
+Say that you used SCIP to solve a number of instances and would like to get the solving time, number of nodes used and the status of the solver for each instance. This short tutorial will show you how to do that with ipet. As an input, it is assumed that you have the output of scip in a separate file for each instance in the folder `SCIP_OUT_FILES`.
+
+### Step 1
```

Comment:
Actually step 1 is not necessary if the files are concatenated afterwards by the new script, because it leaves you with a `concatenated.out` file anyways. 

I mentioned in the concept section that the logfiles that are parsed by ipet need the `.out` extension.